content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -438,6 +438,14 @@ Run the following commands on the virtual machine you want to add to the Istio m
     Hello version: v1, instance: helloworld-v1-578dd69f69-fxwwk
     {{< /text >}}
 
+## Next Steps
+
+For more information about virtual machines:
+
+* [Virtual Machine Architecture](/docs/ops/deployment/vm-architecture/) to learn about the high level architecture of Istio's virtual machine integration.
+* [Debugging Virtual Machines](/docs/ops/diagnostic-tools/virtual-machines/) to learn more about troubleshooting issues with virtual machines.
```

Comment:
The wording of <LINK> to learn more about ... is a bit weird, do we use that elsewhere?

I would have it more like:

- LINK to troubleshoot issues with virtual machines.
- LINK to set up an example deployment of virtual machines.